internal/formats/common/cyclonedxhelpers/format.go

```diff
@@ -137,7 +139,7 @@ func toDependencies(relationships []artifact.Relationship) []cyclonedx.Dependenc
 	for _, r := range relationships {
 		exists := isExpressiblePackageRelationship(r.Type)
 		if !exists {
-			log.Warnf("unable to convert relationship from CycloneDX 1.3 JSON, dropping: %+v", r)
+			log.Debugf("unable to convert relationship from CycloneDX 1.3 JSON, dropping: %+v", r)
```

Comment:
Not sure this was needed or ideal, but the logs were filled with this message.

Reply:
this sounds like it should stay a warning. Is the case that it's hitting common / legitimate?

Reply:
it seems to happen a lot including the tests